src/instruments/src/SD/Pages/Bleed/elements/EngineBleed.tsx

```diff
@@ -25,6 +26,19 @@ const EngineBleed: FC<EngineBleedProps> = ({ x, y, engine, sdacDatum, enginePRVa
     const [precoolerInletPress] = useSimVar(`L:A32NX_PNEU_ENG_${engine}_PRECOOLER_INLET_PRESSURE`, 'psi', 10);
     const precoolerInletPressTwo = Math.round(precoolerInletPress / 2) * 2;
 
+    const [wingAntiIceValveClosed] = useSimVar(`L:A32NX_PNEU_${engine}_WING_ANTI_ICE_VALVE_CLOSED`, 'bool', 500);
+    const wingAntiIceValveClosedBool = wingAntiIceValveClosed === 1;
```

Comment:
You should be able to use the `wingAntiIceValveClosed` value as boolean already if I'm not mistaken